### ISSUE TITLE
Not working with error code 403

### ISSUE THREAD

**Issue by Suresh-16** [vishaltelangre/music-dl#19]
ERROR: Error: Request failed with status code 403
Not downloading any.

**Comment by deepakmahakale** [comment]
@vishaltelangre even I am facing the same issue..
```bash
$ music-dl https://gaana.com/song/seorita-237
Album/Playlist: Señorita (/home/deepak/senorita)
→ (1/1) Señorita
ERROR: Error: Request failed with status code 403
```


I have added the API key as well
```bash
$ music-dl -k AIxxxxxxxxxxxxxxxxxxxxxxxxxxxik
OK: Stored key at: /home/deepak/.music-dl.json
```

**Comment by vishaltelangre** [comment]
Fixed in version `0.0.7`, reinstall using `npm install -g music-dl`.
